hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: minor
+
+This release merges :class:`hypothesis.stateful.Bundle` as subclass of :class:`hypothesis.strategies.SampledFromStrategy`, 
+bringing support for ``consumes(some_bundle).filter(...)`` and improved `filter` and `map` 
+capabilities for stateful tests. (:issue:`3944`)
+
+Thanks to Reagan Lee for this feature!
```

Comment:
The `SampledFromStrategy` class is a private implemenation detail, so I'd write this as:

```suggestion
This release changes :class:`hypothesis.stateful.Bundle` to use the internals of
:func:`~hypothesis.strategies.sampled_from`, improving the `filter` and `map` methods.
In addition to performance improvements, you can now ``consumes(some_bundle).filter(...)``!

Thanks to Reagan Lee for this feature (:issue:`3944`).
```